### PR TITLE
[MT-1088] Update dependencies

### PR DESCRIPTION
**Pull request by Enricoza**

### CODE REVIEW

File: Package.swift [hunk 1]
```diff
@@ -1,26 +1,33 @@
...
+        .tvOS(.v9),
     ],
     products: [
         .library(name: "TealiumCrashModule", targets: ["TealiumCrashModule"])
     ],
     dependencies: [
-        .package(url: "https://github.com/tealium/tealium-swift", from: "2.1.0"),
-        .package(url: "https://github.com/tealium/plcrashreporter", from: "1.5.0")
+        .package(name: "TealiumSwift", url: "https://github.com/tealium/tealium-swift", .upToNextMajor(from: "2.6.0")),
+        .package(name: "PLCrashReporter", url: "https://github.com/microsoft/plcrashreporter", .upToNextMajor(from: "1.10.0")),
```
Comment (trister1997) [inline Package.swift:16]: 2.7.0 was just released, should we go ahead and just update to that?

File: TealiumCrashModule/TealiumCrashModuleTests/test_tealium_crash/TealiumCrashModuleTests.swift [hunk 2]
```diff
@@ -132,4 +132,9 @@ class MockDataLayer: DataLayerManagerProtocol {
     
     func deleteAll() { }
     
+    func add(data: [String : Any], expiry: Expiry) {
+    }
+    
+    func add(key: String, value: Any, expiry: Expiry) {
+    }
```
Comment (trister1997) [inline TealiumCrashModule/TealiumCrashModuleTests/test_tealium_crash/TealiumCrashModuleTests.swift:139]: are these function supposed to be empty?

File: TealiumCrashModuleExample/TealiumCrashModuleExample/ContentView.swift [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+            Button {
+                tealium.track(TealiumEvent("Some Event"))
+            } label: {
+                Text("Track")
+            }.padding()
+            Button {
+                CrashReporter.invokeCrash(name: "asd", reason: "asd")
+//                NSException.raise(NSExceptionName("Exception"),
+//                                  format: "This is a test exception",
+//                                  arguments: getVaList(["nil"]))
```
Comment (trister1997) [inline TealiumCrashModuleExample/TealiumCrashModuleExample/ContentView.swift]: Do we need this commented code?

File: TealiumCrashModuleExample/TealiumCrashModuleExample/ContentView.swift [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+struct ContentView: View {
+    var body: some View {
+        Group {
+            Button {
+                tealium.track(TealiumEvent("Some Event"))
+            } label: {
+                Text("Track")
+            }.padding()
+            Button {
+                CrashReporter.invokeCrash(name: "asd", reason: "asd")
```
Comment (trister1997) [inline TealiumCrashModuleExample/TealiumCrashModuleExample/ContentView.swift]: there might be a better example app crash than `asd` haha

File: Package.swift (hunk 1 above)
Comment (Enricoza) [reply Package.swift:16]: The version 2.6.0 is the minimum version, so 2.7.0 is still included in this release

File: TealiumCrashModule/TealiumCrashModuleTests/test_tealium_crash/TealiumCrashModuleTests.swift (hunk 2 above)
Comment (Enricoza) [reply TealiumCrashModule/TealiumCrashModuleTests/test_tealium_crash/TealiumCrashModuleTests.swift:139]: Yes, the whole class is empty as it's a mock class, and the methods are just added for the interface

File: TealiumCrashModuleExample/TealiumCrashModuleExample/ContentView.swift (hunk 4 above)
Comment (Enricoza) [reply TealiumCrashModuleExample/TealiumCrashModuleExample/ContentView.swift]: definitely 😄

File: TealiumCrashModuleExample/TealiumCrashModuleExample/ContentView.swift (hunk 3 above)
Comment (Enricoza) [reply TealiumCrashModuleExample/TealiumCrashModuleExample/ContentView.swift]: actually the old code is the correct one

**Pull request merged**